Makefile

```diff
@@ -294,7 +299,7 @@ endif
 else
 
 	# Install `7z` into libexec/
-	$(INSTALL_M) $(build_bindir)/7z $(DESTDIR)$(libexecdir)/
+	$(INSTALL_M) $(build_bindir)/7z$(EXE) $(DESTDIR)$(libexecdir)/
```

Comment:
BTW, when `USE_SYSTEM_P7ZIP=1`, this should probably create a symlink to the system's executable, just like we do for libraries? Otherwise we can't recommend users to look for 7z in libexecdir.

Reply:
We can't know where `7z` will be on the host system; there is no single install location that is standard.  Packages that want to support non-default installations of Julia should probably push `Sys.LIBEXECDIR` (I like that suggestion, feel free to open a PR and I'll review it) onto the front of `ENV["PATH"]`, then use `Sys.which()` to search for `7z`.

Reply:
Wouldn't pushing libexecdir in front of of `PATH` go against the idea that this folder shouldn't interfere with other users? Callers could just try `$(Sys.LIBEXECDIR)/7z`, and use `Sys.which` only if that fails.

TBH that's really not a priority for me right now (I just came across this on Slack).

Reply:
I don't mean push it system-wide or even Julia-process-wide; I mean something like:

```
7z_path = withenv("PATH" => string(Sys.LIBEXECDIR, Sys.iswindows() ? ";" : ":", ENV["PATH"]) do
	Sys.which(`7z`)
end
```